spec/features/users/user_updates_info_spec.rb

```diff
@@ -33,26 +34,39 @@
     fill_in 'Email', with: ""
     click_button('Update')
 
-    expect(page).to have_content("Email can't be blank")
+    expect(page).to have_content("can't be blank")
   end
 
   scenario 'user is notified of successful update' do
     click_button('Update Info')
 
     fill_in 'Email', with: "foo@bar.com"
+    fill_in 'Current password', with: @user.password
     click_button('Update')
 
-    expect(page).to have_content("Profile updated")
+    expect(page).to have_content("Your account has been updated successfully.")
   end
 
-  scenario 'user is taken to user show page after successful update' do
+  scenario 'user is taken to dinosaur index after successful update' do
     click_button('Update Info')
 
     fill_in 'Email', with: "foo@bar.com"
+    fill_in 'Current password', with: @user.password
+    click_button('Update')
+
+    expect(current_path).to eq(root_path)
+  end
+
+  scenario "user changes profile photo" do
+    click_button('Update Info')
+
+    attach_file 'Profile Photo',
+      "#{Rails.root}/spec/support/images/example_photo.jpg"
+    fill_in 'Current password', with: @user.password
+
     click_button('Update')
 
-    expect(page).to have_content("foo@bar.com")
-    expect(current_path).to eq(user_path(@user))
+    expect(page).to have_content("Your account has been updated successfully.")
   end
 
 end
```

Comment:
We probably want to test somewhere that the profile image is in the page: `expect(page).to have_css("img[src*=name-of-image.jpg']")`